SelfAndPeerEvaluations/utils.py

```diff
@@ -0,0 +1,439 @@
+import canvasapi
+from collections import defaultdict
+import csv
+from collections import deque
+import json
+import statistics
+import random
+
+
+class PartnerEvalQuizIndividualStats:
+    def __init__(self, name: str, id: int, files: list):
+        self.name = "".join(filter(str.isalnum, name))
+        self.id = str(id)
+        self.scores = defaultdict(list)
+
+        self.detailed_scores = []
+        self.qualitative_subjects = None
+
+        self.csv_file_path = (
+            f"./quiz_results/individual_student_reports/{self.name}.csv"
```

Comment:
The path can't be hard coded. Needs to be able to specified as an argument to the __init__ method